site/docs/tools/deltalake/spark.md

```diff
@@ -0,0 +1,191 @@
+# Spark via Delta Lake
+
+!!! note
+    Detailed steps on how to set up Pyspark + Delta Lake + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-delta-demo-nba.ipynb).
+
+To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
+the [Spark 2](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark2/{{ versions.java}}/nessie-deltalake-spark2-{{ versions.java}}.jar)
+or [Spark 3](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark3/{{ versions.java}}/nessie-deltalake-spark3-{{ versions.java}}.jar)
+jar. These jars contain all Nessie **and** Delta Lake libraries required for operation.
+
+!!! note
+    the `spark3` jar is for Delta versions >0.7.0 on Spark3 and the `spark2` jar is for Delta versions 0.6.x on Spark2.4
+
+In pyspark this would look like
+
+``` python
+SparkSession.builder
+    .config('spark.jars.packages',
+            'org.projectnessie:nessie-deltalake-spark3:{{ versions.java}}')
+    ... rest of spark config
+    .getOrCreate()
+```
+
+The Nessie LogStore needs the following parameters set in the Spark/Hadoop config.
+
+```
+nessie.url = full url to nessie
+nessie.username = username if using basic auth, omitted otherwise
+nessie.password = password if using basic auth, omitted otherwise
+nessie.auth.type = authentication type (BASIC, NONE or AWS)
```

Comment:
Please change to the latest property names (per #1968), i.e. `nessie.authentication.username`, `nessie.authentication.password`, `nessie.authentication.type`

Reply:
However, it might be preferable to use `nessie.authentication.type=NONE` in docs by default and add a separate page for non-trivial auth settings (those will be the same across all systems, I think). WDYT?

Reply:
Sorry I missed it, I will add it shortly. Regarding your question, honestly, I have no strong opinion since I am still 'yet' not fully into the subject :). Thus, if you think  `nessie.authentication.type=NONE` would be better , then I can do that :)

Reply:
Let's keep existing examples "as is" for this PR since it's mostly just a refactoring. I'll revisit them later for #1981.

Reply:
I've just merged #1968... please rename properties accordingly within current examples. I'll tackle auth-specific docs under #1981.

Reply:
@dimas-b I have resolved the conflicts and renamed the auth properties.